src/main/java/org/opentripplanner/middleware/otp/response/Itinerary.java

```diff
@@ -246,4 +246,31 @@ public long getScheduledEndTimeEpochMillis() {
         }
         return endTimeEpochMillis;
     }
+
+    /**
+     * Returns true if the current time falls between the start and end time of the itinerary
+     */
+    public boolean isActive() {
```

Comment:
We might need the Json/Bson Ignore annotations here. (I don't think we need them on `hasEnded`.)

Reply:
Added in d95910c71758df3212e5f878a91a509f02839dd0.